lstchain/calib/camera/drs4.py

```diff
@@ -83,11 +84,14 @@ def _fill_pedestal_event_jit(waveform, expected_pixel_id, first_cap_array,
                             numped[gain, pixel, posads] += 1
 
     def finalize_pedestal(self):
-        if np.sum(self.numped==0) > 0:
-            raise RuntimeError("Not enough events to coverage all capacitor. "
-                               "Please use more events to create pedestal file.")
+        self.meanped = self.meanped / self.numped
+        pixels_with_nan_value = np.unique(np.where(np.isnan(self.meanped) == True)[1])
```

Comment:
`== True` not needed, `np.isnan` already returns an array of booleans.

Reply:
and indices to arrays are by definition unique. I think you mean something different here.

How about checking that at least one of the gains is nan:

```
pixels_with_nan_value, = np.where(np.isnan(meanped).any(axis=0))  # any of the gain channels has a nan
```